docs/website/docs/general-usage/customising-pipelines/removing_columns.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Removing columns
+description: Removing columns by passing list of column names
+keywords: [deleting, removing, columns, drop]
+---
+
+# Removing columns
+
+Removing columns before loading data into a database is a reliable method to eliminate sensitive or
+unnecessary fields. For example, in the given scenario, a source is created with a "country_id" column,
+which is then excluded from the database before loading.
+
+Let's create a sample pipeline demonstrating the process of removing a column.
+
+1. Create a source function that creates dummy data as follows:
+
+   ```python
+   import dlt
+
+   @dlt.source
+   def dummy_source():
+       # This function creates a dummy data source.
+       @dlt.resource(write_disposition='replace')
+       def dummy_data():
+
+             for i in range(3):
+               yield {'id': i, 'name': f'Jane Washington {i}', 'country_code': 40 + i}
+       return dummy_data()
+   ```
+   This function creates three columns `id`, `name` and `country_code`.
+
+1. Next, create a function to filter out columns from the data before loading it into a database as follows:
+
+   ```python
+   from typing import Dict, List, Optional
+
+   def remove_columns(doc: Dict, remove_columns: Optional[List[str]] = None) -> Dict:
+
```

Comment:
black code format doesn't approve empty lines in the very first line of the function

Reply:
I would recommend you use black (shed) for code snippets as well and look how it's changing, this way your code style skills will improve constantly

Reply:
used black, shed, mypy and flake8 ))